app/main.py

```diff
@@ -1,3 +1,68 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, int):
+            return Distance(self.km + other)
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+
+    def __iadd__(self, other):
+        if isinstance(other, Distance):
+            self.km += other.km
+        if isinstance(other, int):
+            self.km += other
+
+        return self
+
+    def __mul__(self, other):
+        if isinstance(other, int):
+            return Distance(self.km * other)
```

Comment:
$27 not sure if this is necessary, but we need to multiply only by `int` the same for $32`__truediv__`